MemberExpectations.md

```diff
@@ -0,0 +1,58 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the code of conduct but that supports
+the vision, mission and operating principles of the Node.js project.
+
+While members have the right and responsibility to exercise independent
+judgment and to express dissenting opinions during discussions within
+the organization, members also have the obligation outside the
+organization (twitter, reddit, etc.) to respect and support decisions
+and values of the majority, even when a member dissented from
+the majority view. A member who does not support a team decision
+or current team value may express the member's opposition within
+the team in an appropriate manner, but must not take actions publicly
+that have the purpose or result of undermining the decisions, values
+or actions of the team. Accordingly, a member who itends to publicly
+oppose a team decision action should resign from the team before doing so.
+
+In addition to being willing to act as a team player, members of our
+leadership groups must conduct themselves in a professional and
+respectful manner. Some general
+guidelines include:
+
+- Serve as ambassadors of the vision, mission and operating
+  principles of the Node.js project.
+- Treat all community members as professionals. with respect, consideration,
+  and valuing a diversity of views and opinions. Strive to avoid preferential
+  treatment, and hold everyone (including ourselves) accountable to the same
+  set of standards.  Everyone gets to speak up. 
+- Deal with issues directly with the person in question. Resist complaining
+  about others in the project in a public sphere.
+- Keep your promises. Your word is your bond. Commit to anything you can
+  deliver upon. 
+- Be the model of accountability and leadership. Provide the example of
+  ownership and stewardship that everyone can follow to success.
+- Commit to ongoing development and learning best practices for governing.
+- Critique ideas rather than individuals, discussing any concerns in person
+  whenever possible, and  taking responsibility for our statements by
+  speaking as much as possible
+  in the first person (.I. statements) rather than in the third person.
+- Collectively and as an individual serve as a role model to demonstrate
+  the highest standards of ethical conduct.
+- Remediate quickly when you realize you made a mistake. Leaders are human,
+  and they will make mistakes - however they should act swiftly to
+  acknowlege mistakes and correct them. Most often this can mitigate
+  any damage done.
+- Aim to remediate first and then discuss.  If other members of the
+  team express concerns about actions, acknowledge their concerns by
+  stopping the actions in question and then discuss within the team
+  to come to a common agreement.
```

Comment:
This one should be at the top as well

Reply:
Moved,although I'm a bit worried it leads off with a bit of a negative tone.